keyboards/handwired/3dp660_oled/info.json

```diff
@@ -0,0 +1,246 @@
+{
+    "manufacturer": "xia0",
+    "keyboard_name": "handwired/3dp660_oled",
+    "maintainer": "xia0",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6"],
+        "rows": ["D5", "B0", "B5", "B4", "E6", "D7", "C6", "D4", "D2", "D3"]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x3661",
+        "vid": "0x1A00"
```

Comment:
This is in use already: 
https://www.the-sz.com/products/usbid/index.php?v=0x1A00&p=&n=

Ideally, the further past 0x4000, the better.